recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "0.14.2" %}
+{% set version = "0.14.3" %}
 
 package:
   name: panel
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/panel/panel-{{ version }}.tar.gz
-  sha256: c22690833dfa508d72ae632e2d42ea7d63ee2a31347e23b7ecf8dcded312ec74
+  sha256: 61d1949f0bad6894bad1c0ea0b3b0782c15ef94dd780671da2c3f0f4f461b547 
 
 build:
   number: 0
```

Comment:
`bokeh 2.4.3` in `host` would be preferable, see https://github.com/holoviz/panel/blob/d0e1fe5b55b4d1c90026d58b4260b76170e5a6dd/pyproject.toml#L6. It's the latest version in the branch **2.x.x**.
I'm not sure if other pinnings should be changed.